code/modules/station_goals/dna_vault.dm

```diff
@@ -257,8 +257,10 @@ var/list/non_simple_animals = typecacheof(list(/mob/living/carbon/monkey,/mob/li
 	switch(upgrade_type)
 		if(VAULT_TOXIN)
 			H << "<span class='notice'>You feel resistant to airborne toxins.</span>"
-			S.tox_breath_dam_min = 0
-			S.tox_breath_dam_max = 0
+			if(/obj/item/organ/lungs in H.internal_organs)
```

Comment:
false.

the ```H.internal_organs``` list will never contain the path ```/obj/item/organ/lungs```, or any paths at all for that matter!
It's a lit of objects.